internal/cmd/index/db_test.go

```diff
@@ -100,52 +101,58 @@ INSERT INTO person_caseitem (person_id, caseitem_id) VALUES (1, 1), (1, 2);
 		}
 	}()
 
-	err = r.queryByID(ctx, resultsCh, 1, 2, 10, "person_1", person.AliasName)
-	if assert.Nil(err) && assert.Len(results, 2) {
+	err = r.queryByID(ctx, resultsCh, 1, 3, 10, "person_1", person.AliasName)
+
+	// wait a bit for the async func above to catch up :/
+	time.Sleep(time.Second)
```

Comment:
instead of turning the channel into an array we could pull each value from the channel and compare, then wouldn't need to sleep